libraries/AP_TemperatureSensor/AP_TemperatureSensor_MLX90614.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include "AP_TemperatureSensor_Backend.h"
+
+#if AP_TEMPERATURE_SENSOR_MLX90614_ENABLED
+
+#define MLX90614_I2CDEFAULTADDR 0x5A    // Device default slave address 
+#define MLX90614_BROADCASTADDR  0       // Device broadcast slave address 
+#define MLX90614_CRC8POLY       7       // CRC polynomial = X8+X2+X1+1 
+#define MLX90614_XDLY           25      //  Experimentally determined delay to prevent read
+
+//  RAM addresses
```

Comment:
Most of the defines here are only relevant for the implementation and could be moved to the .cpp file.

Also, most of them are unused.... should they be used?

Reply:
There are options to take ambience temperature or object temperature readings. For them there are different registers address from where we can pull reading so I taken them but not used. I have only used ambience temperature.

`#define MLX90614_TA                0x06    //  RAM reg - Linearized temperature, ambient 
#define MLX90614_TOBJ1          0x07    //  RAM reg - Linearized temperature, source #1 
#define MLX90614_TOBJ2          0x08    //  RAM reg - Linearized temperature, source #2 
`